include/nova_renderer/constants.hpp

```diff
@@ -0,0 +1,10 @@
+//! \brief A collection of constants for Nova to use
+
+#pragma once
+
+#include <string>
+
+namespace nova::renderer {
+    const std::string ModelMatrixBufferName = "NovaModelMatrixBuffer";
+    const std::string PerFrameDataName = "NovaPerFrameUBO";
```

Comment:
Variable naming seems a bit wrong here. I think this should either be `SCREAMING_SNAKE_CASE` or `snake_case`.

Reply:
Fixed